pipeline/filters/lua.md

```diff
@@ -197,11 +208,11 @@ We want to extract the `sandboxbsh` name and add it to our record as a special k
 {% tab title="fluent-bit.conf" %}
```

Comment:
We probably need YAML equivalent as well.

Reply:
do you want to submit it?